### PR TITLE
Feature/fes scanned mr04

### PR DESCRIPTION
**Pull request by danfalc**

### CODE REVIEW

File: src/test/java/uk/gov/companieshouse/pageobjects/ConfirmationStatementPage.java [hunk 1]
```diff
@@ -48,5 +58,30 @@ public ConfirmationStatementPage completeNoUpdateCs01() {
...
+        String nextCsDue = elementNextCsDue.getText();
+        elementCsDateInput.sendKeys(nextCsDue);
+        elementPaymentReceivedYes.click();
+        completeCapitalTab();
+        saveForm();
+        return this;
+    }
+
+    /**
+     * Do capital tab.
```
Comment (rsuller) [inline src/test/java/uk/gov/companieshouse/pageobjects/ConfirmationStatementPage.java:76]: I know we've probably got loads of comments similar to this, but they're not exactly the most helpful for those that may be new to using this.. I think we should aim to be a little more descriptive where we can. Just a thought mind.

File: src/test/resources/uk/gov/companieshouse/features/cs01.feature [hunk 2]
```diff
@@ -43,3 +43,9 @@ different filing method is necesssary.
       | org_unit                      | country          |
       | Scottish DEB (FES)            | Scotland         |
       | NI Document Examination (FES) | Northern Ireland |
+
+  Scenario: Accepted paper CS01 form
```
Comment (rsuller) [inline src/test/resources/uk/gov/companieshouse/features/cs01.feature:47]: This one is probably petty. Just shifting the `When` step (action) to the actual completion of the form.
```
 Given I am logged in as a user in the "RM1" organisational unit
 And I process the start document for form "CS01"
 When I complete mandatory details to file a paper confirmation statement
 Then company history information is updated with the accepted CS01 transaction
```

**Review by rsuller — APPROVED** [review]

File: src/test/java/uk/gov/companieshouse/pageobjects/ConfirmationStatementPage.java (hunk 1 above)
Comment (danfalc) [reply src/test/java/uk/gov/companieshouse/pageobjects/ConfirmationStatementPage.java:76]: Lazy copy and paste from the old stuff from me. good spot

File: src/test/resources/uk/gov/companieshouse/features/cs01.feature (hunk 2 above)
Comment (danfalc) [reply src/test/resources/uk/gov/companieshouse/features/cs01.feature:47]: Agree

**Pull request merged**